definitions/tools/docm_add_variants.cwl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "CombineVariants (GATK 3.6)"
+baseCommand: ["/usr/bin/java", "-Xmx8g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 9000
+      tmpdirMin: 25000
+    - class: DockerRequirement
+      dockerPull: mgibio/cle
+arguments:
+    ["-genotypeMergeOptions", "PRIORITIZE",
+     "--rod_priority_list", "callers,docm",
```

Comment:
Does this mean we lose the list of callers? I know in several projects where the number of callers, ex. 3 of 4, was used as a post-pipeline filter.

Reply:
Nope!  see /gscmnt/gc2708/info/medseq/tmp/toiltest/output3/cromwell-executions/detect_variants.cwl/ab14a12b-b095-42e9-af4d-afa00cbca514/call-annotated_filter_index/execution/annotated_filtered.vcf.gz for an example

Reply:
awesome! thanks!

Reply:
to further expound - docm variants will never overwrite existing variants, only get added where there is no variant,  so the original line (with all caller info) will remain.